test/NamingStrategy/MapNamingStrategyTest.php

```diff
@@ -40,6 +41,12 @@ public function invalidKeyValues() : iterable
         yield 'float'      => [1.1];
     }
 
+    public function testConstructorIsPrivate()
+    {
+        $this->expectException(Error::class);
+        new MapNamingStrategy([], []);
```

Comment:
This is covered by PHPStan - wouldn't need an explicit test for it

Reply:
How is it covered by phpstan? We don't have code otherwise that attempts to exercise the constructor, and this is _new_ behavior...

Reply:
@weierophinney visibility of methods, properties, etc is type information: type information is checked by type checkers, not by runtime tests :+1:

Reply:
Okay, removed.